packages/animated-circle/index.android.ts

```diff
@@ -1,8 +1,17 @@
-import { Color } from '@nativescript/core';
+import { Color, Property, booleanConverter } from '@nativescript/core';
 import { AnimatedCircleCommon, barColorProperty, rimColorProperty, spinBarColorProperty } from './common';
 
 declare const at;
 
+export const animatedProperty = new Property({
```

Comment:
We'll need types on this one. If types worked as expected, your build would complain about `updateAnimatedCircle` being private.
Take this one as an example: https://github.com/NativeScript/plugins/blob/main/packages/keyboard-toolbar/common.ts#L13

Then, we can mark `updateAnimatedCircle` as public and rename it to `_updateAnimatedCircle`.